src/runtime_src/driver/xclng/drm/xocl/lib/libqdma/qdma_compat.h

```diff
@@ -45,17 +45,6 @@
 
 #endif
 
-/* use simple wait queue (swaitq) with newer kernels */
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 6, 0)
```

Comment:
@karenx-xilinx  Could you help to review this qdma file?

Reply:
The simple waitqueue (i.e., swait_xxx) is preferred in qdma, which is available in v5.0. Please keep them included instead of reverting back to the normal waitqueue.

Reply:
@keryell , could you address @karenx-xilinx 's comment?

Reply:
@karenx-xilinx Do you mean we can cherry-pick more from the `master` branch?